### ISSUE TITLE
OPAT Test version - Line

### ISSUE THREAD

**Issue by 7575colli** [openhealthcare/elcid#502]
Line:
The day counter should count the day that the line was inserted as day ‘0’.  It currently counts this as day ‘1’

Can the ‘external length’ be changed to ‘external length when inserted’

When ‘peripheral cannula’ is added, a ‘removal time’ field appears.  This is currently set to mandate that a date is added instead of a time. 
When a cannula is added, the ‘external length’ field rightly disappears, but the ‘lists’ display still displays ‘(cm)’ – Can that be erased please.  

Complications – please add look up list:
None
Phlebitis
Bacteraemia
Exit site infection
Thrombophlebitis 
Blocked
Dislodged 

Removal reason – Please add look up list:
IV therapy discontinued
Routine replacement required  
Complication

**Comment by michaeledwardmarks** [comment]
Checking through this on Heruko:

- Complications – Already has look-up list, for example try typing "blockage"
- Removal reason – this list already contains  both "No longer required" and "Routine Replacement"

Not sure I follow this one
- When ‘peripheral cannula’ is added, a ‘removal time’ field appears. This is currently set to mandate that a date is added instead of a time. 


Agree with @7575colli that the below seem reasonable. Not sure how many of them definitely need doing before go live.

- [ ]The day counter should count the day that the line was inserted as day ‘0’. It currently counts this as day ‘1’

- [ ] Can the ‘external length’ be changed to ‘external length when inserted’. 

- [ ] When a cannula is added, the ‘external length’ field rightly disappears, but the ‘lists’ display still displays ‘(cm)’ – Can that be erased please.

**Comment by 7575colli** [comment]
The issues i raised above were from Heroku app.

Iv just had a look on the UCLH test version and found the following issues:

1) When 'IV Cannula' is entered into the lines field, the screen doesnt switch to allow me to enter the time that the line was inserted.
2) No look up list for 'Site'

Regarding:
•When ‘peripheral cannula’ is added, a ‘removal time’ field appears. This is currently set to mandate that a date is added instead of a time

I cant test this on the UCLH version because i currently cant add a time that the line was inserted/removed.
Basically, when i tried to enter a time, i couldn’t.  Instead, i was presented with a calendar

**Comment by michaeledwardmarks** [comment]
Ok. These are all essentially because of the lookup list issue have already documented in #514 

I think if you type Peripheral cannula in then the time field will appear - it does for me.
Once we port the lookup-lists over I can make IV cannula a synonym of Peripheral canula which will fix this.

**Comment by 7575colli** [comment]
Your quite right.  I got it to do the same.  

Did you note that the 'removal time' field brings up a calendar?

**Comment by michaeledwardmarks** [comment]
Noted.

 - Removal date brings up calendar (Correct)
- [ ] Removal time should be formatted HHMM - identical to Inserted Time

**Comment by michaeledwardmarks** [comment]
@davidmiller 
Same here as with antimicrobials.
Not sure these OPAT updates have deployed

**Comment by davidmiller** [comment]
It's now deployed to elcid-search (Yes I still mean elcid-search :)

**Comment by michaeledwardmarks** [comment]
Fixed.
